app/helpers/application_helper.rb

```diff
@@ -15,19 +15,9 @@ def duration_display(minutes)
     hours = minutes / 60
     minutes = minutes - (hours * 60)
 
-    minute_string = minutes > 0 ? "#{minutes}m" : ""
-    hour_string = hours > 0 ? "#{hours}h" : ""
+    minute_string = minutes > 0 ? "#{minutes}" : ""
+    hour_string = hours > 0 ? "#{hours}" : ""
 
-    "#{hour_string} #{minute_string}".strip
-  end
-
-  def long_duration_display(minutes)
-    hours = minutes / 60
-    minutes = minutes - (hours * 60)
-
-    minute_string = minutes > 0 ? "#{minutes} #{'minute'.pluralize(minutes)}" : ""
-    hour_string = hours > 0 ? "#{hours} #{'hour'.pluralize(hours)}" : ""
-
-    "#{hour_string} #{minute_string}".strip
+    "#{hour_string}:#{minute_string}".strip
```

Comment:
@J3RN `Fixnum` doesn't contain the `format` method, but it is on `Kernel`. The change I pushed up calls `format('%02d', hours)` instead.